console/network/src/canary_v0.rs

```diff
@@ -503,6 +506,11 @@ impl Network for CanaryV0 {
     ) -> bool {
         path.verify(&*CANARY_POSEIDON_4, &*CANARY_POSEIDON_2, root, leaf)
     }
+
+    /// Returns the height at which a specified consensus version becomes active.
+    fn HEIGHT_V(version: usize) -> Result<u32> {
+        Ok(Self::CONSENSUS_VERSIONS.get(version).ok_or(anyhow!("Invalid consensus version"))?.0)
```

Comment:
since we're indexing the versions starting from `1`, shouldn't this instead be the following?
```suggestion
        Ok(Self::CONSENSUS_VERSIONS.get(version - 1).ok_or(anyhow!("Invalid consensus version"))?.0)
```
The input `version` could be a `NonZeroUsize` to avoid any confusion about the beginning.

Reply:
with such a setup the tuples would IMO be redundant (assuming that the versions are going to always be contiguous)

Reply:
Thx, should be fixed by c928f35

Reply:
Thx, should be fixed by c928f35